projects/lexical_credo/lib/lexical_credo.ex

```diff
@@ -0,0 +1,102 @@
+defmodule LexicalCredo do
+  alias Lexical.Document
+  alias Lexical.Plugin.Diagnostic
+  alias Lexical.Project
+
+  use Diagnostic, name: :lexical_credo
+  require Logger
+
+  def init do
+    with {:ok, _} <- Application.ensure_all_started(:credo) do
+      :ok
```

Comment:
We don't need the CLI output, so I think we can call the `    GenServer.call(Credo.CLI.Output.Shell, {:suppress_output, true})` here